pages/app/models/refinery/page.rb

```diff
@@ -164,20 +150,20 @@ def nullify_duplicate_slugs_under_the_same_parent!
     end
 
     def translated_to_default_locale?
-      persisted? && translations.any?{ |t| t.locale == Refinery::I18n.default_frontend_locale}
+      persisted? && translations.any?{ |t| t.locale.to_sym == Refinery::I18n.default_frontend_locale}
```

Comment:
Nice catch! Mobility doesn't convert translation locales to symbols like Globalize does.

Reply:
Added [a note about this](https://github.com/shioyama/mobility/wiki/Migrating-from-Globalize#translation-locale-is-a-string-not-a-symbol) to the Mobility wiki page on migrating from Globalize.